geoportailv3/models.py

```diff
@@ -17,6 +17,10 @@
 Base = sqlahelper.get_base()
 
 
+LayerWMTS.__acl__ = [DENY_ALL]
+WMTSDimension.__acl__ = [DENY_ALL]
```

Comment:
@sbrunner the changelog indicates that the same should be done for `LayerInternalWMS.__acl__ ` and `LayerExternalWMS.__acl__`. Should I do it as well? I wonder because these two classes are extended in the case of the Luxembourg project.